docusaurus/docs/recipes/dropdown.md

```diff
@@ -0,0 +1,427 @@
+
+⚠️ If you want to control yourself some of these state pieces, you can pass them as prop. E.g.: `useSelect({ items, isOpen: true })`. This example would force the menu to be always open.
+
+_Prop Getters_
+
+One of the core concepts of this library is _prop getters_ such as `getToggleButtonProps()` and `getMenuProps()`. These functions must be applied to the proper element and then Downshift will take care of wiring things up and make them work properly. Thus, `useSelect` provides us the following [_prop getters_](https://github.com/downshift-js/downshift/tree/master/src/hooks/useSelect#prop-getters):
+- `getToggleButtonProps` - a function that returns the props must be applied to the menu toggle button element. We are applying it to our `button` element. Further details on their [documentation](https://github.com/downshift-js/downshift/tree/master/src/hooks/useSelect#gettogglebuttonprops).
+- `getItemProps` - a function that returns the props must be applied to any menu item element. We are applying it to our `item` element. Further details on their [documentation](https://github.com/downshift-js/downshift/tree/master/src/hooks/useSelect#getitemprops).
+- `getLabelProps` - a function that returns the props must be applied to the label element. Further details [here](https://github.com/downshift-js/downshift/tree/master/src/hooks/useSelect#getlabelprops).
+- `getMenuProps` - a function that returns the props must be applied to the menu's root element. Further details [here](https://github.com/downshift-js/downshift/tree/master/src/hooks/useSelect#getmenuprops).
+
+_Actions_
+
+Although not used in this example, there is a set of props, called _actions_, returned by this hook that can be very useful to change its state imperatively. Here are some examples:
+
+- `toggleMenu` - toggle the menu open state
+- `openMenu` - opens the menu
+- `reset` - resets downshift's state
+
+ℹ️ You can see the full list of _actions_ [here](https://github.com/downshift-js/downshift/tree/master/src/hooks/useSelect#actions).
+
+#### 2. Component Usage
+
+Here is an example of how to use the `<Dropdown>` component we've built so far:
+
+```js
+import React, { useCallback } from 'react';
+import items from './items';
+import Dropdown from './Dropdown';
+
+export default function App() {
+  const handleStateChange = useCallback((state) => console.log(state), []);
+
+  return (
+    <div>
+      <h2>Dropdown with Downshift 🏎</h2>
+      <Dropdown items={ items } onStateChange={ handleStateChange } />
+    </div>
+  );
+}
+```
+
+#### 3. Native _select_ behavior support
+
+On mobile/touch devices the use of native _select_ behavior it's highly recommended due to UX reasons. To implement that, our `<Dropdown>` component is going to need some enhancements. Our approach will place a native select element on top of our trigger button, with the same items that we are passing to our `<Dropdown>` component, but it will invisible.   
```

Comment:
```suggestion
On mobile/touch devices the use of native _select_ behavior it's highly recommended due to UX reasons. To implement that, our `<Dropdown>` component is going to need some enhancements. Our approach will place a native select element on top of our trigger button, with the same items that we are passing to our `<Dropdown>` component, but it will be invisible.   
```